<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PagedTableView</class>
 <widget class="QWidget" name="PagedTableView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>352</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>511</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout" stretch="0,0">
    <item>
     <widget class="PagedTableBar" name="controlBar" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PagedTableBar</class>
   <extends>QWidget</extends>
   <header location="global">presenter/widgets/PagedTableBar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
